kernel/sched.c

```diff
@@ -501,15 +512,8 @@ static k_spinlock_key_t slice_expired_locked(k_spinlock_key_t sched_lock_key)
 }
 
 /* Called out of each timer interrupt */
-void z_time_slice(int ticks)
-{
-	/* Hold sched_spinlock, so that activity on another CPU
-	 * (like a call to k_thread_abort() at just the wrong time)
-	 * won't affect the correctness of the decisions made here.
-	 * Also prevents any nested interrupts from changing
-	 * thread state to avoid similar issues, since this would
-	 * normally run with IRQs enabled.
-	 */
```

Comment:
Sorta specious comment got removed.  Not strictly related, the lock is still present (for the obvious reasons mentioned)